lib/sensu-plugins-supervisor/version.rb

```diff
@@ -1,8 +1,8 @@
 module SensuPluginsSupervisor
   module Version
     MAJOR = 1
-    MINOR = 0
-    PATCH = 3
+    MINOR = 1
```

Comment:
feel free to tell me to remove this and make it part of the release but it made testing easier and I will release this after it gets merged.